Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp

```diff
@@ -166,5 +184,31 @@ namespace AZ
             m_rpiSystem.UnregisterXRSystem();
         }
 
+        AZ_CVAR_EXTERNED(AZ::u32, r_metricsNumberOfCaptureBatches);
+        AZ_CVAR_EXTERNED(AZ::CVarFixedString, r_metricsDataLogType);
+        AZ_CVAR_EXTERNED(AZ::u32, r_metricsWaitTimePerCaptureBatch);
+        AZ_CVAR_EXTERNED(AZ::u32, r_metricsFrameCountPerCaptureBatch);
+
+        void RPISystemComponent::InitializePerformanceCollector()
+        {
+            auto onBatchCompleteCallback = [](AZ::u32 pendingBatches) {
+                AZ_TracePrintf("RPISystem", "Completed a performance batch, still %llu batches are pending", pendingBatches);
```

Comment:
`AZ_TracePrintf` doesn't output a newline automatically. One should be added.
```suggestion
                AZ_TracePrintf("RPISystem", "Completed a performance batch, still %llu batches are pending\n", pendingBatches);
```

Reply:
Fixed.